pkg/storaged/pages.jsx

```diff
@@ -710,12 +710,12 @@ export const PageTable = ({ emptyCaption, aria_label, pages, crossrefs, sorted,
                     { pages &&
                     <Thead>
                         <Tr>
-                            { show_icons && <Th aria-label={_("Category")} />}
-                            <Th>{_("ID")}</Th>
-                            <Th>{_("Type")}</Th>
-                            <Th>{_("Location")}</Th>
-                            <Th className="storage-size-column-header">{_("Size")}</Th>
-                            <Th aria-label={_("Actions")} />
+                            { show_icons && <Td aria-hidden="true" /> }
```

Comment:
As far as I can tell, this is not a "hack" but completely valid.

I looked at docs for thead, tr, td, th. It seems there's nothing stopping anyone from putting either a td or th inside of a parent element of tr, even if the parent of that tr is a thead or tbody (or in some cases, even just table, if there is no caption, thead, and tbody).